ethereumetl/jobs/extract_geth_traces_job.py

```diff
@@ -51,6 +54,10 @@ def _extract_geth_traces(self, geth_traces):
         for geth_trace_dict in geth_traces:
             geth_trace = self.geth_trace_mapper.json_dict_to_geth_trace(geth_trace_dict)
             traces = self.trace_mapper.geth_trace_to_traces(geth_trace)
+
+            calculate_trace_statuses(traces)
```

Comment:
Geth traces don't have transaction_hash in them so status calculation is not correct. `calculate_trace_statuses` needs to be accommodated to geth tracds